go/internal/connector_api_error.go

```diff
@@ -26,7 +26,10 @@ func ParseConnectorApiError(response []byte) error {
 	if err != nil {
 		return &wrapError{
 			outer: fmt.Errorf("failed to: %s", ACTION_JSON_UNMARSHAL),
-			inner: err,
+			inner: &wrapError{
+				inner: err,
+				outer: fmt.Errorf("%s", response),
+			},
```

Comment:
This is so if the error returned by the API is not parsable, we still get the "raw" response in the logs rather than an ambiguous "Failed to parse json"